hudi-client/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -77,6 +77,9 @@
   public static final String BLOOM_INDEX_INPUT_STORAGE_LEVEL = "hoodie.bloom.index.input.storage.level";
   public static final String DEFAULT_BLOOM_INDEX_INPUT_STORAGE_LEVEL = "MEMORY_AND_DISK_SER";
 
+  public static final String BLOOM_INDEX_SHOULD_UPDATE_PARTITION_PATH = "hoodie.bloom.index.should.update.partition.path";
```

Comment:
should we include "global" in the name. Bcoz, this is applicable only incase of global. In case of regular, the new record will be inserted as a new entry to new partition path.

Reply:
Also, can we add some java docs since the property is not very intuitive.

Reply:
Renamed and added docs.